tests/roles/ovn_adoption/handlers/main.yaml

```diff
@@ -0,0 +1,2 @@
+- name: stop adoption helper pod
+  command: oc delete pod adoption --ignore-not-found=true
```

Comment:
We should allow for "debug mode" where the pod would stay around if the test suite fails, or maybe even just if the user wants it to. But this one is not a blocker and can be done in a different PR.

Reply:
ack, let's do in follow ups

Reply:
done in the updated revision